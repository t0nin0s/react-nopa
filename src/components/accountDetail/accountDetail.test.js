import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import AccountDetail from './accountDetail'

const account = {
  account: 'Natwest',
  name: "Doe J.",
  accountNumber: "123456789",
  sortCode: "600028"
}

test('AccountDetail component renders as expected', () => {
  const component = shallow(<AccountDetail />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
