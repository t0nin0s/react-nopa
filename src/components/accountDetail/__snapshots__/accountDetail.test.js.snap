// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountDetail component renders as expected 1`] = `
<div
  className="container"
>
  <div
    className="details left"
  >
    <h4>
      Natwest
    </h4>
    <h4>
       Doe J.
    </h4>
  </div>
  <div
    className="details right"
  >
    <h4>
      Current Account
    </h4>
    <h4>
      12345698
    </h4>
    <h4>
      600028
    </h4>
  </div>
</div>
`;
